src/dns/packet.rs

```diff
@@ -60,6 +61,13 @@ impl<'a> PacketBuilder<'a> {
         self
     }
 
+    /// Adds an answer to the packet with class IN
+    pub fn add_answer_in(&mut self, name: &'a str, ttl: Duration, data: RData<'a>) -> &mut Self {
+        self.answers.push(ResourceRecord { name, ttl, class: Class::IN, data });
+        self.header.an_count += 1;
+        self
+    }
+
```

Comment:
I'm not so sure about this method, but I agree that the ergonomics could be better, I have something else in mind for this.

Reply:
Yeah can't say i was a big fan of it either, I've removed it for now and pushed the changes.